patches/minecraft/net/minecraft/world/food/FoodData.java.patch

```diff
@@ -0,0 +1,20 @@
+--- a/net/minecraft/world/food/FoodData.java
++++ b/net/minecraft/world/food/FoodData.java
+@@ -24,9 +_,15 @@
+       this.f_38697_ = Math.min(this.f_38697_ + (float)p_38708_ * p_38709_ * 2.0F, (float)this.f_38696_);
+    }
+ 
++   @Deprecated
+    public void m_38712_(Item p_38713_, ItemStack p_38714_) {
+-      if (p_38713_.m_41472_()) {
+-         FoodProperties foodproperties = p_38713_.m_41473_();
++      this.eat(p_38713_, p_38714_, null);
++   }
++
++   public void eat(Item item, ItemStack stack, @javax.annotation.Nullable net.minecraft.world.entity.LivingEntity entity)
```

Comment:
I feel like this overloaded eat method should have the item and stack use the srg names and then just be added to forge's exc file so that they can be properly mapped. That way if more things that use them get added later in the method at some point they don't need to be replaced via patch to reference these names.